code/modules/cargo/exports.dm

```diff
@@ -122,26 +119,39 @@ Credit dupes that require a lot of manual work shouldn't be removed, unless they
 // Called only once, when the object is actually sold by the datum.
 // Adds item's cost and amount to the current export cycle.
 // get_cost, get_amount and applies_to do not neccesary mean a successful sale.
-/datum/export/proc/sell_object(obj/O, contr = 0, emag = 0)
-	var/the_cost = get_cost(O)
+/datum/export/proc/sell_object(obj/O, datum/export_report/report, dry_run = TRUE, allowed_categories = EXPORT_CARGO , apply_elastic = TRUE)
+	var/the_cost = get_cost(O, allowed_categories , apply_elastic)
 	var/amount = get_amount(O)
-	total_cost += the_cost
+
+	if(amount <=0 || the_cost <=0)
+		return FALSE
+	
+	report.total_value[src] += the_cost
+	//total_cost += the_cost
```

Comment:
?

Reply:
No idea why i left this commented. Will remove in a moment.